src/openzeppelin/mocks/reentrancy_mock.cairo

```diff
@@ -0,0 +1,85 @@
+#[abi]
+trait IReentrancyGuardAttacker {
+  fn call_sender();
+}
+
+#[abi]
+trait IReentrancyGuard {
+    fn count_this_recursive(n: felt252);
+}
+
+#[contract]
+mod ReentrancyMock {
+    // OZ modules
+    use openzeppelin::security::reentrancyguard::ReentrancyGuard;
+    use openzeppelin::utils::check_gas;
+
+    // Dispatchers
+    use super::IReentrancyGuardAttackerDispatcher;
+    use super::IReentrancyGuardAttackerDispatcherTrait;
+    use super::IReentrancyGuardDispatcher;
+    use super::IReentrancyGuardDispatcherTrait;
+
+    // Other
+    use starknet::ContractAddress;
+    use starknet::get_caller_address;
+    use starknet::get_contract_address;
+
+    struct Storage {
+        _counter: felt252
+    }
+
+    #[constructor]
+    fn constructor(initial_number: felt252) {
+        _counter::write(initial_number);
+    }
+
```

Comment:
do we need a constructor and a parameter? can't we just rely on default behavior of `felt252`?